zk-keygen/src/main.rs

```diff
@@ -178,6 +307,22 @@ impl KeyType {
     }
 }
 
+#[derive(Debug, Error)]
+#[error("unsupported key type: \"{0}\"")]
```

Comment:
```suggestion
#[error("unexpected key type specifier: \"{0}\"")]
```

so as to not imply that they specified a string that _could_ be supported